pages/linux/toolbox-run.md

```diff
@@ -0,0 +1,16 @@
+# toolbox run
+
+> Run a command in an existing toolbox container.
+> More information: <https://www.mankier.com/1/toolbox-run>.
+
+- Run command inside a toolbox container with the given NAME:
+
+`toolbox run --container {{NAME}}`
```

Comment:
This applies to all your pages:

- ``` `toolbox` ```, instead of `toolbox`
- All arguments like `NAME` should be in snake case (`name`)

Reply:
I will update it now.